xenit-aps/values.yaml

```diff
@@ -4,174 +4,28 @@ general:
-      readOnly: true
-    - mountPath: >-
-        /usr/local/tomcat/shared/classes/alfresco/extension/subsystems/Authentication/ldap-ad/oup-ad2
-      name: ldap2-ad-auth-volume
-      readOnly: true
-    - mountPath: >-
-        /usr/local/tomcat/shared/classes/alfresco/extension/subsystems/Authentication/ldap-ad/oup-ad3
-      name: ldap3-ad-auth-volume
-      readOnly: true
-  volumes:
-    - configMap:
-        defaultMode: 420
-        items:
-          - key: ldap-ad-authentication.properties
-            path: ldap-ad-authentication.properties
-        name: ldap1-ad-auth-config
-      name: ldap1-ad-auth-volume
-    - configMap:
-        defaultMode: 420
-        items:
-          - key: ldap-ad-authentication.properties
-            path: ldap-ad-authentication.properties
-        name: ldap2-ad-auth-config
-      name: ldap2-ad-auth-volume
-    - configMap:
-        defaultMode: 420
-        items:
-          - key: ldap-ad-authentication.properties
-            path: ldap-ad-authentication.properties
-        name: ldap3-ad-auth-config
-      name: ldap3-ad-auth-volume
-  livenessProbe:
-    httpGet:
-      path: /alfresco/api/-default-/public/alfresco/versions/1/probes/-live-
-      port: 8080
-      scheme: HTTP
-    failureThreshold: 1
-    initialDelaySeconds: 130
-    periodSeconds: 20
-    successThreshold: 1
-    timeoutSeconds: 10
-  readinessProbe:
-    httpGet:
-      path: /alfresco/api/-default-/public/alfresco/versions/1/probes/-ready-
-      port: 8080
-      scheme: HTTP
-    failureThreshold: 6
-    initialDelaySeconds: 60
-    periodSeconds: 20
-    successThreshold: 1
-    timeoutSeconds: 10
-digitalWorkspace:
-  enabled: true
-  replicas: 1
-  image:
-    registry: 'quay.io'
-    repository: 'alfresco/alfresco-digital-workspace'
-    tag: '3.0.0'
-  strategy:
-    type: RollingUpdate
-  resources:
-    requests:
-      memory: "256Mi"
-      cpu: "150m"
-  basePath: "/workspace"
 
-share:
+postgresqlActiviti:
   enabled: true
-  mergeAcsShare: false
-  replicas: 1
   image:
     registry: 'docker.io'
-    repository: 'xenit/alfresco-share-community'
-    tag: '7.3'
+    repository: 'xenit/postgres'
+    tag: 'latest'
   strategy:
     type: RollingUpdate
   resources:
     requests:
-      memory: "256Mi"
-      cpu: "0.5"
-  livenessProbe:
-    failureThreshold: 1
-    initialDelaySeconds: 130
-    periodSeconds: 20
-    successThreshold: 1
-    timeoutSeconds: 10
-  readinessProbe:
-    failureThreshold: 6
-    initialDelaySeconds: 60
-    periodSeconds: 20
-    successThreshold: 1
-    timeoutSeconds: 10
-
-mq:
-  enabled: true
-  replicas: 1
-  image:
-    registry: 'docker.io'
-    repository: 'alfresco/alfresco-activemq'
-    tag: '5.18.3-jre17-rockylinux8@sha256:25386b20263b7e838605e07fea2713fb65762010c1c677cf82aecefbaed5d227'
-  strategy:
-    type: Recreate
-  resources:
-    requests:
-      memory: "512Mi"
-      cpu: "0.5"
-
-postgresql:
+      memory: "1Gi"
+      cpu: "1"
+postgresqlActivitiAdmin:
```

Comment:
If this is a DB pod that requires persistent volumes the strategy.type should be Recreate.